Jenkinsfile

```diff
@@ -81,6 +81,109 @@ pipeline {
+                        cd src/pfe/portal
+                        npm install
+                        npm run eslint
+                        if [ $? -ne 0 ]; then
+                            exit 1
+                        fi
+                        cd ../../..
+
+                        # Create codewind-workspace if it does not exist
+                        printf "\n\nCreating codewind-workspace\n"
+                        mkdir -m 777 -p codewind-workspace
+
+                        export REPOSITORY='';
+                        export TAG
+                        export WORKSPACE_DIRECTORY=$PWD/codewind-workspace;
+                        export HOST_OS=$(uname);
+                        export REMOTE_MODE;
+                        export HOST_HOME=$HOME
+                        export ARCH=$(uname -m);
+                        # Select the right images for this architecture.
+                        if [ "$ARCH" = "x86_64" ]; then
+                          export PLATFORM="-amd64"
+                        else
+                          export PLATFORM="-$ARCH"
+                        fi
+
+                        # Start codewind running
+                        ~/docker-compose -f docker-compose.yaml -f docker-compose-local.yaml up -d;
+
+                        if [ $? -eq 0 ]; then
+                            # Reset so we don't get conflicts
+                            unset REPOSITORY;
+                            unset WORKSPACE_DIRECTORY;
+                            unset REMOTE_MODE;
+                            printf "\n\n${GREEN}SUCCESSFULLY STARTED CONTAINERS $RESET\n";
+                            printf "\nCurrent running codewind containers\n";
+                            docker ps --filter name=codewind
+                        else
+                            printf "\n\n${RED}FAILED TO START CONTAINERS $RESET\n";
+                            exit;
+                        fi
+
+                        printf "\n\nPausing for 20 seconds to allow containers to start\n";
+                        sleep 20;
+
+                        # Check to see if any containers exited straight away
+                        printf "\n\n${BLUE}CHECKING FOR codewind CONTAINERS THAT EXITED STRAIGHT AFTER BEING RUN $RESET\n";
+                        EXITED_PROCESSES=$(docker ps -q --filter "name=codewind" --filter "status=exited"  | wc -l)
+                        if [ $EXITED_PROCESSES -gt 0 ]; then
+                          printf "\n${RED}Exited containers found $RESET\n";
+                          # docker ps --filter "name=codewind" --filter "status=exited";
+                          NUM_CODE_ZERO=$(docker ps -q --filter "name=codewind" --filter "status=exited" --filter "exited=0" | wc -l);
+                          NUM_CODE_ONE=$(docker ps -q --filter "name=codewind" --filter "status=exited" --filter "exited=1" | wc -l);
+                          if [ $NUM_CODE_ZERO -gt 0 ]; then
```

Comment:
Should this be `-eq` instead of `-gt`?

Reply:
These are the flags we are currently using in the build and test scripts (not made any changes)

Reply:
but we want to check for having 1 or more exited containers